tensorflow_probability/examples/jupyter_notebooks/Linear_Mixed_Effects_Models.ipynb

```diff
@@ -590,11 +588,17 @@
     "    ])\n",
     "\n",
     "    # To enable tracking of the trainable variables via the created distribution,\n",
-    "    # we attach a reference to `self`. Since all TFP objects sub-class\n",
-    "    # `tf.Module`, this means that the following is possible:\n",
-    "    # LinearMixedEffectModel()(features_train).trainable_variables\n",
-    "    # ==> tuple of all tf.Variables created by LinearMixedEffectModel.\n",
-    "    model._to_track = self\n",
+    "    # we attach a reference to `self`. Since all TFP objects (e.g. JointDistributionSequential in this case )\n",
+    "    # sub-class `tf.Module`, this means that if attach an object of \n",
+    "    # type tf.Module (i.e. LinearMixedEffectModel in this case) to it then the variables (i.e tf.Variables created\n",
```

Comment:
change "it" to something like "the return value", and no need to use the parenthetical here. What do you think of something like 

```
# Since all TFP objects (e.g., `JointDistributionSequential` here) sub-class 
# `tf.Module`, this means that if we attach an object of type `tf.Module` 
# (the `LinearMixedEffectModel` in this case) to the return value, then the 
# `tf.Variables` created by `LinearMixedEffectModel` will be automatically 
# tracked. Practically this next line allows:
#
# LinearMixedEffectModel()(features_train).trainable_variables
# ==> tuple of all tf.Variables created by LinearMixedEffectModel
```

Reply:
ok. looks better.